package/package.json

```diff
@@ -136,6 +136,7 @@
     "react-dom": "17.0.2",
     "react-native": "0.64.1",
     "react-native-builder-bob": "0.18.1",
+    "react-native-fs": "2.18.0",
```

Comment:
This is already a peer dependency (maybe not from us but from one of our dependencies and the guide tells users to install it)